### DISCUSSION TITLE
Stepper motor specs Sovol Sv06

### DISCUSSION THREAD

**Discussion by Heiko910** [bassamanator/everything-sovol-sv06#39] [General]
I was wondering if anyone has the specs for the stepper motors of a Sovol SV06? I saw in the Bassamanator repository "everything-Sovol-SV06" that he has the specs listed, but I noticed on a parts supplier webpage as well as on my printer, that the X and Y stepper share a part # and the z axis steppers are a different #.
The physical dimensions seem to be the same.
Here are the part numbers from my printer.
X and Y axis: steppers: 42BYGH1634-A-20Db
Z axis stepper: 42BYGH1603-A-20DN

Does anyone know what the difference is?
Thanks,
Heiko

**Comment by bassamanator** [comment]
To get the answers you seek, you will have to contact the makers of the stepper motors themselves. Part numbers do not have to adhere to any international standard, though a manufacturer will probably have some standardized naming convention for the motors (or other parts) that they build.

Marlin users who don't want to compile their own firmware, if you want to replace your motors, you need to match:
- current
- step angle

Motor size (e.g., Nema **17**) and height are important regardless of the firmware, because the motor needs to fit into place.

Klipper users can simply change the above mentioned requirements in their `printer.cfg` and be good to go.

I imagine the steppers (not including the extruder motor) are generating ~25N.cm of torque?
